test/routes/eligibility-checker/how-certain.route.test.js

```diff
@@ -80,31 +80,31 @@ describe('/eligibility-checker/how-certain route', () => {
       const element = document.querySelector(`#${elementIds.help1}`)
       expect(element).toBeTruthy()
       expect(TestHelper.getTextContent(element)).toEqual(
-        'To use this service, you must be completely certain.'
+        'If you know your item\'s eligible you can continue to either:'
       )
     })
 
     it('should have the correct help text 2', () => {
       const element = document.querySelector(`#${elementIds.help2}`)
       expect(element).toBeTruthy()
       expect(TestHelper.getTextContent(element)).toEqual(
-        'If you’re still unsure, we can help you decide.'
+        'If you\'re not sure your item\'s eligible, you can answer a series of questions to help you find out.'
       )
     })
 
     it('should have the correct radio buttons', () => {
       TestHelper.checkRadioOption(
         document,
         elementIds.howCertain,
-        'Completely',
-        'Completely'
+        'Yes, I know my item\'s eligible',
+        'Yes, I know my item\'s eligible'
       )
 
       TestHelper.checkRadioOption(
         document,
         elementIds.howCertain2,
-        'I’d like some help to work this out',
-        'I’d like some help to work this out'
+        'I need help to find out',
+        'I need help to find out'
```

Comment:
As a random aside, I find these route tests pretty pointless - they just check whether you've typed the same copy text into this test as you've typed into the code ☹️ 

Nothing you can do about it at this point, but I feel it sort of defeats the purpose of having unit tests in the first place.